{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","text":"AWSErrorContext"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/sotocore\/awserrortype\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorType\/context","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"additional context information related to the error"}],"kind":"symbol","metadata":{"role":"symbol","title":"context","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:8SotoCore12AWSErrorTypeP7contextAA0C7ContextVSgvp","required":true,"modules":[{"name":"SotoCore"}]},"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorType"]]},"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"role":"symbol","title":"AWSErrorContext","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSErrorContext"}],"abstract":[{"type":"text","text":"Additional information about error"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"url":"\/documentation\/sotocore\/awserrorcontext"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorType/context":{"role":"symbol","title":"context","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"additional context information related to the error"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorType\/context","kind":"symbol","required":true,"type":"topic","url":"\/documentation\/sotocore\/awserrortype\/context"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorType":{"role":"symbol","title":"AWSErrorType","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSErrorType"}],"abstract":[{"type":"text","text":"Standard Error type returned by Soto."}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorType","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AWSErrorType"}],"url":"\/documentation\/sotocore\/awserrortype"},"doc://codes.soto.soto-core/documentation/SotoCore":{"role":"collection","title":"SotoCore","abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","kind":"symbol","type":"topic","url":"\/documentation\/sotocore"}}}